src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -139,18 +139,17 @@ public async Task<IActionResult> OnGetAsync(string id, string revisionId = null)
 
             if (!String.IsNullOrEmpty(ReviewContent.ActiveAPIRevision.Files.First().CrossLanguagePackageId))
             {
-                var correspondingReviewId = await _apiRevisionsManager.GetReviewIdsOfLanguageCorrespondingReviewAsync(ReviewContent.ActiveAPIRevision.Files.First().CrossLanguagePackageId);
-                var correspondingReviews = await _reviewManager.GetReviewsAsync(reviewIds: correspondingReviewId.Where(_ => _ != id).ToList(), isClosed: false);
-                foreach (var review in correspondingReviews)
+                var correspondingAPIRevisions = await _apiRevisionsManager.GetLanguageCorrespondingAPIRevisionsAsync(ReviewContent.ActiveAPIRevision.Files.First().CrossLanguagePackageId);
```

Comment:
Won't this fetch all API revisions with same package ID? What will happen once all new revisions for a given package is created with this ID? what's the logic to identify the exact revision from this list.

Reply:
Yes, I thought of this. We can let the user select the APIRevision or find a way to default, maybe the latest automatic.

Reply:
Can't we filter the revisions using associated PR number on top of cross language package Id to find the exact revision generated from same PR?